docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '2'
+
+services:
+
+
+    jekyll:
+        build:
+            context: jekyll/docs
+        command:
+            - jekyll serve --watch --incremental --force_polling
+        volumes:
+            - ./jekyll/docs:/srv/jekyll
+        ports:
+            - "4000:4000"
+
+    web:
+        image: sosedoff/pgweb
+        environment:
+            - DATABASE_URL=postgres://codefordc:codefordc@postgres:5432/codefordc_postgres_docker?sslmode=disable
+        links:
+            - postgres
+        ports:
+            - "8081:8081"
+        restart: "on-failure:10"
+
+    postgres:
+        #housing-insights-postgres is an empty Postgres 9.5 install
+        image: codefordc2/housing-insights-postgres
+        environment:
+            - POSTGRES_PASSWORD=codefordc
+            - POSTGRES_USER=codefordc
+            - POSTGRES_DB=codefordc_postgres_docker
+        ports:
+            - "5432:5432"
+        restart: "on-failure:3"
+
+    sandbox:
```

Comment:
I think we should remove this from the top level compose... unless we're going to provide an example API as part of the tool suite.

Reply:
My thinking was that we could make a top-level compose that is configured to run a container for every sub folder. So if we add ruby on rails, node.js, etc. as additional folders we'd add them to the top level compose. 

This sandbox can be used for running a Python-driven api, but it can also be used for running arbitrary Python scripts. The port mapping just isn't used if you don't serve something onto the port.

Reply:
Also, in the future improvements section, I'd be happy to gut the housing insights Flask API and turn it into a boilerplate example to include here.

Reply:
👍 on a top level docker compose, thats a really neat idea.